packages/init2/src/lib/repositoryDomain.ts

```diff
@@ -0,0 +1,121 @@
+const ABC123 = "abcdefghijklmnopqrstuvwxyz0123456789";
+
+// 11 characters long or less adjectives
+const ADJECTIVES = [
+	"adorable",
+	"beautiful",
+	"charming",
+	"cloudy",
+	"delightful",
+	"focused",
+	"honest",
+	"inspiring",
+	"lovable",
+	"melodious",
+	"pleasant",
+	"pretty",
+	"shiny",
+	"sugary",
+	"tall",
+	"valliant",
+].filter((adjective) => adjective.length <= 11);
+
+// 11 characters long or less bakeries
+const BAKERIES = [
+	"baguette",
+	"briouat",
+	"crepe",
+	"croissant",
+	"donut",
+	"eclair",
+	"flaons",
+	"kolompeh",
+	"macaron",
+	"paste",
+	"pretzel",
+	"rustico",
+	"semla",
+	"strudel",
+	"taiyaki",
+	"toast",
+].filter((bakery) => bakery.length <= 11);
```

Comment:
I wonder if the `filter()` could be accomplished using a TypeScript type, one that only accepts words less than 12 characters.

Reply:
![image](https://user-images.githubusercontent.com/25330882/205611796-724f01d6-8427-4e13-8089-7553e1d07cd1.png)

I tried this way, but TypeScript says no ^^'

Reply:
Haha, oh well!